### PR TITLE
Fixdpc8

### PR DESCRIPTION
**Pull request by u-fischer**
Hopefully resolves https://github.com/davidcarlisle/dpctex/issues/8.

### CODE REVIEW

File: required/tools/longtable.dtx [hunk 1]
```diff
@@ -1223,7 +1225,7 @@
 %    \begin{macrocode}
 \def\LT@start{%
   \let\LT@start\endgraf
-  \endgraf\penalty\z@\vskip\LTpre
+  \endgraf\penalty\z@\vskip\LTpre\endgraf
```
Comment (FrankMittelbach) [inline required/tools/longtable.dtx]: I think the additional \endgraf is weird enough to deserve an explanation, after all we are in vertical mode at this point. This is to get the page builder executed I presume? If so th should be mentioned.

Comment (u-fischer) [reply required/tools/longtable.dtx]: @FrankMittelbach  The problem is that in the next line \dimen@ is set to \pagetotal but without the \endgraf this doesn't take the \LTpre into account as \pagetotal is updated only after the \endgraf and so the calculation is wrong.

Comment (FrankMittelbach) [reply required/tools/longtable.dtx]: I understand (this is triggering the page builder as I assumed) but I think it should be mentioned/documented that it is there for this reason

**Comment by FrankMittelbach** [comment]
your call. I decided that I'm not adding anything in unless it was distributed as -dev or is a hotfix that is just needed (or is a fix for a -dev issue).

**Comment by davidcarlisle** [comment]
> 
> 
> your call. I decided that I'm not adding anything in unless it was distributed as -dev or is a hotfix that is just needed (or is a fix for a -dev issue).

I wouldn't do the other lt fixes at this time, but this is just an \endgraf that usually does nothing and in the cases where it changes the behaviour the current code is clearly wrong, so I'm minded to merge (probably by hand rather than accept this pr  and resolving the conflicts in the tlmgr setup

**Comment by davidcarlisle** [comment]
Main change to lt updated from this branch to develop and change log updated there so closing this pr

**Pull request closed without merging**